automatic/sysinternals/tools/chocolateyInstall.ps1

```diff
@@ -21,6 +21,23 @@ if (Is-NanoServer) {
   $packageArgs.checksum = '7b6fab7f0ada0bc2b0748f3bc29e93846869f9dd175a0ca070bbebc07ee5de09'
  }
 
+if (-Not $pp.NoStartMenu) {
```

Comment:
Since the package do not already have the ability to create start menu icons, this parameter should be inverted.
So if a user wants the start menu icons, they should opt-in for them.

Reply:
I pondered this a bit. I finally decided to put it this way, because it is a pretty common parameter: https://www.google.com/search?q=site%3Achocolatey.org%2Fpackages%2F+%2FNoStartMenu
I assume somehow using `chocolateyBeforeModify.ps1` to avoid the entries sudden popping up is not an option, right?

Reply:
True, it is quite common for packages to have the `NoStartMenu` parameter, however, most (if not all) of these packages already added start menu icons (from the installer), or are completely new packages.
This is not a new package, and a start menu icon has not been created before.
So, as to not suddenly surprise users already used to install the package and maybe do not want the start menu icons, or have already their own implementation of creating the icons this is something that will need to be an opt-in, instead of an opt-out.